features/get-all-items-breadcrumbs.feature

```diff
@@ -84,30 +108,45 @@ Scenario: Corrupt breadcrumb hierarchy (one parent-child link missing), but user
     | 43 | 13      | 23     | null            | true              | false                | false               | 0             | 0        |
   And the database has the following table 'items_items':
     | ID | idItemParent | idItemChild | iChildOrder | iDifficulty | iVersion |
-    | 51 | 21           | 22          | 1           | 0           | 0        |
     | 52 | 22           | 23          | 1           | 0           | 0        |
   And I am the user with ID "1"
   When I send a GET request to "/items/?ids=21,22,23"
-  Then the response code should be 200
+  Then the response code should be 400
   And the response body should be, in JSON:
     """
-    [
-    { "item_id": 21, "language_id": 1, "title": "Graph: Methods" },
-    { "item_id": 22, "language_id": 1, "title": "DFS" },
-    { "item_id": 23, "language_id": 1, "title": "Reduce Graph" },
-    { "item_id": 21, "language_id": 2, "title": "Graphe: Methodes" }
-    ]
+    {"success":false,"message":"Bad Request","error_text":"The IDs chain is corrupt"}
```

Comment:
Side note: The same it has been done for `add-item`, I would prefer moving the error handling case to a different file (`get-all-items-breadcrumbs.robustness.feature`). This way it is easier to reflect on all the happy paths (all scenarios) on one side, and on all possible errors handled on the other.  
As there are other error cases already in this file, we can do this once it has been merged.  
The same way, the error handling should be using the step "the response error message should contain ..." instead of exact error matching.